.circleci/config.yml

```diff
@@ -81,6 +81,18 @@ executors:
     machine:
       image: ubuntu-1604:202004-01
     resource_class: xlarge
+  test-runner:
+    # Config for cluster on EKS/GKE
+    environment:
+      # Set cluster workes to 5 (5 cluster nodes)
+      NUM_NODES: 4
+      NUM_WORKERS: 5
+      ENTERPRISE_IMAGE_NAME: splunk/splunk:edge
```

Comment:
we are using "edge" here. Confirming we always test our develop branch with Splunk edge image?

Reply:
Good point @kashok-splunk . Since edge will have the latest ansible scripts, a good idea to run nightly against them so that we can catch any failures early